java/dynamodb-tools/src/main/java/sleeper/dynamodb/tools/DynamoDBAttributes.java

```diff
@@ -88,6 +96,18 @@ public static Instant getInstantAttribute(Map<String, AttributeValue> item, Stri
         return buildInstant.apply(Long.parseLong(string));
     }
 
+    public static Optional<Double> getDoubleAttribute(Map<String, AttributeValue> item, String key) {
```

Comment:
I think we should probably be consistent about either returning boxed values with an optional, or unboxed values with a default parameter.

I think it's probably best to return them unboxed to avoid unnecessary boxing/unboxing.